getting_started/cloud_infrastructure_access/dedicated-aws-access.adoc

```diff
@@ -0,0 +1,56 @@
+[id="dedicated-aws-access"]
+= AWS Infrastructure Access
+include::modules/common-attributes.adoc[]
+:context: dedicated-aws-access
+
+AWS infrastructure access allows link:https://access.redhat.com/node/3610411[Customer Portal Organization Admins] and cluster owners to enable Amazon Web Services (AWS) Identity and Access Management (IAM) users to have federated access to the AWS Console for their {product-title} cluster. Admins may select between Network Management or Read-only access options.
+
+[id="dedicated-configuring-aws-access"]
+== Configuring AWS Infrastructure Access
+
+.Prerequisites
+. An existing AWS account. You may create a new AWS account by following link:https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/[AWS documentation].
+
+. An IAM user with `STS:AllowAssumeRole` permissions within the AWS account.
```

Comment:
It does seem odd to have this lengthy procedure in a prereq. I wonder if you could make the prereqs a separate procedure. Something like:

```
== Sign up for an AWS account with IAM permissions
//Be sure to add an intro sentence.
.Procedure
. Set up AWS IAM permissions
Before you can configure access to AWS infrastructure, you will need to set up IAM permissions in your AWS account.
1. Create a new AWS account...
2. Configure IAM user permissions with `STS:...` 
3. etc.

== Configure AWS Infrastructure Access
.Procedure
1. Open the...
```